io.github.TTimo.GtkRadiant.appdata.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is not specific to Flathub; you should maintain it in your repo instead.